dashboard/config/overlays/ci-openbsd-pf/etc/rc.conf.local

```diff
@@ -0,0 +1,10 @@
+# Overrides the minimal config in the image created by
+# tools/create-openbsd-vmm-worker.sh to enable PF.
+cron_flags=NO
+library_aslr=NO
+ntpd_flags=NO
+pf=YES
```

Comment:
While we are here are there any other things to turn on? I would say [almost] anything that enables additional subsystems should be enabled. @mptre

Reply:
These are mostly userspace features that's I'm disabling for faster OS boot. I don't know any we want to re-enable. I would'be happy to pare down the default config in create-openbsd-vmm-worker.sh if @mptre can tell me where I went overboard.